CHANGELOG.md

```diff
@@ -1,3 +1,21 @@
+# 1.10.1 (2023-10-12)
+
+### Features
+
+#### stdlib
+
+- Support Android API levels 24 - 27 ([#13884](https://github.com/crystal-lang/crystal/pull/13884), thanks @HertzDevil)
```

Comment:
This one should probably go into bug fixes because it's only a regression in 1.10.0 that these api levels were violated